packages/@rnw-scripts/doxysaurus/src/doc-model.ts

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ **/
+
+//
+// Documentation model.
+//
+
+// Documentation model for a code project.
+export class DocModel {
```

Comment:
If intent here, below, is to describe type of these models instead of providing an implementation, would be better to export a type or interface than class.

E.g.
```ts
export type DocModel = {
  codeProject: string;
  ...
}
```
This allows multiple representations. E.g. object literals, classes, etc.

Reply:
If class is to allow default values, consider just letting those be optional. That also makes the distinction clearer between empty string vs undefined. TS will give you compiler warnings about undefined as well.